.drone.yml

```diff
@@ -58,3 +58,29 @@ pipeline:
     when:
       branch: [staging]
       event: [push]
+
+  # tagging
+
+  build_stg:
+    image: golang:1.9-alpine3.7
+    environment:
+      - REACT_APP_SERVER_URL=//code-annotation-staging.srcd.run
+    commands:
+      - apk --update upgrade
+      - apk add --no-cache
+          bash make curl git
+          ca-certificates build-base
+          libxml2-dev protobuf
+          yarn
+      - make prepare-build
+      - make packages
+    debug: true
+    when:
+      event: tag
```

Comment:
Bad format. It should be enclosed in []

Reply:
I copied from http://plugins.drone.io/drone-plugins/drone-github-release

Reply:
It may work, but the rest of the actions in the drone file (and others we're building) are enclosed in brackets, so in order to be consistent please use the same event action format. Thx!

Reply:
Sure, done